packages/ui/src/components/BufficornImage.vue

```diff
@@ -1,26 +1,50 @@
 <template>
-  <div class="bufficorn-image-container">
-    <div v-if="name" class="small-title item">
+  <div class="bufficorn-image-container" :class="{ horizontal }">
+    <div v-if="horizontal" class="small-title item">
       <p class="bufficorn-name">{{ name }}</p>
     </div>
     <img
-      class="bufficorn-image"
+      v-if="ranchName"
+      loading="lazy"
       :class="{ flip }"
-      :src="importPng('Bufficorn-0')"
-      alt="icon"
+      class="ranch-icon"
+      :src="importSvg(RANCHES[ranchName])"
+      alt="Bufficorn"
     />
+    <picture>
+      <source media="(max-width: 799px)" :srcset="importPng(`${name}480w`)" />
+      <source media="(min-width: 800px)" :srcset="importPng(`${name}800w`)" />
+      <source media="(min-width: 369px)" :srcset="importPng(`${name}`)" />
+      <source media="(max-width: 368px)" :srcset="importPng(`${name}100w`)" />
+      <img
+        class="bufficorn-image"
+        :class="{ flip, horizontal }"
+        :src="importPng(name)"
+        :srcset="
+          `${importPng(`${name}100w`)} 100w, ${importPng(
+            `${name}`
+          )} 120w, ${importPng(`${name}480w`)} 480w, ${importPng(
+            `${name}800w`
+          )} 800w`
+        "
+      />
+    </picture>
```

Comment:
![image](https://user-images.githubusercontent.com/44392885/153022649-61e33cb8-f584-407e-ac07-9a973e22957a.png)

The logo is covering the handheld trait